docs/blog/mirror_intro.md

```diff
@@ -0,0 +1,43 @@
+# Funding Open Source to Empower Open Source
+
+> Radicle is software that empowers collaboration on and funding of open source work. The Grants Program is a first step towards many things. (a) Open sourcing the development of Radicle itself (b) sustainability funding the growth of the Radicle product and (c) the start of a new model for funding open source software, dog-fooded by Radicle. 
+> 
+> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+## How do I apply for a grant?
+There are two tracks for applying...
+
+### 🌱 Seed Grants
+
+These are smaller (<$50,000) or more technical grants.
+
+###[Application form here]()
+
+The application will only be shared with members of the grants committee. Applications that pass may be saved in one of our repos.
+
+### 🌲 Tree Grants
+
+These are larger (>$50,000) or more technical grants. 
+
+Copy and fill out out our handy [markdown template here](https://raw.githubusercontent.com/radicle-dev/radicle-grants/main/grants/applications/template.md).
+
+Post that application in the [grants category of Radicle's forum](). 
```

Comment:
@abbey-titcomb on line 24, I'd like to hyperlink to Grants category on Discourse